apps/compute-cf.yml.j2

```diff
@@ -63,7 +73,70 @@ Resources:
 
             --==BOUNDARY==--
 
-  ComputeEnvironment:
+  SharedComputeEnvironment:
```

Comment:
I think we'll actually want to avoid renaming existing resources, as this will probably cause them to be deleted and re-created during the stack deployment, which could disrupt running jobs. (I agree that the `Shared` prefix makes it more readable, but probably not worth the operational disruption.)